src/app/modules/main/communities/controller.nim

```diff
@@ -465,10 +467,20 @@ proc runSignFlow(self: Controller, pin, path, dataToSign: string) =
   self.connectKeycardReponseSignal()
   self.keycardService.startSignFlow(path, dataToSign, pin)
 
+proc hashMessage(self: Controller, message: string): string =
+  try: 
+    let response = status_go.hashMessage(message)
+    let jsonResponse = parseJson(response)
+    return jsonResponse{"result"}.getStr()
+  except Exception as e:
+    error "hashMessage: failed to parse json response", error = e.msg
+    return ""
```

Comment:
This function should be in a service. Could even be a util

Reply:
I think there's no service for this, it's a util function indeed. 
Moved to `backend/general.nim`. Or do you think there's a better place?